README.md

```diff
@@ -446,17 +446,35 @@ cartridge_failover_params:
 
 **Note** that stateful failover is supported since `Cartridge` 2.1.2.
 
+**Note** that `etcd2` provider is supported since `Cartridge` 2.2.0.
+
 `stateful` failover requires these parameters:
 
 - `state_provider`(`string`, required for `stateful` mode) - external state
-  provider type. Now there's only one supported type - `stateboard`.
+  provider type. Supported `stateboard` and `etcd2` providers.
 
 - `stateboard_params`(`dict`, required for `stateboard` state provider) -
   configuration for stateboard:
     - `uri`(`string`, required) - stateboard instance URI;
 
     - `password`(`string`, required) - stateboard instance password;
 
+- `etcd2_params`(`dict`, required for `etcd2` state provider) -
+  configuration for stateboard:
+    - `prefix`(`string`, required) - prefix used for etcd keys: `<prefix>/lock` and
+      `<prefix>/leaders`;
```

Comment:
Default value in cartridge is "/"

Reply:
Btw it's optional in cartridge.

Reply:
Now `etcd2_params` and all it's fields are optional.